### PR TITLE
feat: update circle swap demo to mainnet

### PR DESCRIPTION
**Pull request by StephenFluin**

### CODE REVIEW

File: pages/dev/general-message-passing/_meta.json [hunk 1]
```diff
@@ -10,5 +10,5 @@
     "monitoring": "Monitoring",
     "solidity-utilities": "Solidity Utilities",
     "axelar-sandbox": "Sandbox",
-    "example-composable-usdc": { "display": "hidden"}
+    "example-usdc-cctp": "USDC CCTP"
```
Comment (milapsheth) [inline pages/dev/general-message-passing/_meta.json:13]: add a redirect? this link was sent out publicly before

**Comment by npty** [comment]
There's a 404 link to cross-chain usdc page at the bottom of this page https://axelar-docs-git-stephen-update-for-cctp-axelar-network.vercel.app/dev/general-message-passing/examples

**Comment by npty** [comment]
not sure why this happens, but I can't find the link to the usdc cctp page in the preview link.

**Comment by StephenFluin** [comment]
Vercel + Nextra build caches never work and end up providing an old version
of the navigation. Nav changes require a rebuild without cache. :(

**Review by npty — APPROVED** [review]
LGTM 🚀

**Review by milapsheth — APPROVED** [review]

**Pull request merged**